src/Microsoft.FeatureManagement.AspNetCore/AspNetCoreFeatureManagementBuilderExtensions.cs

```diff
@@ -14,6 +15,21 @@ namespace Microsoft.FeatureManagement
     /// </summary>
     public static class AspNetCoreFeatureManagementBuilderExtensions
     {
+        /// <summary>
+        /// Adds the Microsoft.Targeting feature filter to the service collection along with the <see cref="ITargetingContextAccessor"/> implementation that the filter will use to access the targeting context.
+        /// </summary>
+        /// <typeparam name="T">The <see cref="ITargetingContextAccessor"/> implementation type.</typeparam>
+        /// <param name="builder">The feature management builder to extend.</param>
+        /// <returns>The feature management builder.</returns>
+        public static IFeatureManagementBuilder AddTargetingFilter<T>(this IFeatureManagementBuilder builder) where T : ITargetingContextAccessor
```

Comment:
It's a convenience function but it also hides some details that could cause confusion. Since it's only one extra line, maybe we can just do it as part of the example?

Reply:
Removed and readme updated as well.